src/servers/src/mysql/helper.rs

```diff
@@ -159,7 +159,7 @@ pub fn convert_value(param: &ParamValue, t: &ConcreteDataType) -> Result<ScalarV
             ConcreteDataType::String(_) => Ok(ScalarValue::Utf8(Some(
                 String::from_utf8_lossy(b).to_string(),
             ))),
-            ConcreteDataType::Binary(_) => Ok(ScalarValue::LargeBinary(Some(b.to_vec()))),
+            ConcreteDataType::Binary(_) => Ok(ScalarValue::Binary(Some(b.to_vec()))),
```

Comment:
I'm afraid that we also have other places that map the binary type to `ScalarValue::LargeBinary`. I found several occurrences via global searching `ScalarValue::LargeBinary`.

Reply:
hm... I thought I've covered all of them with my recent changes

Reply:
Mainly in this file
https://github.com/GreptimeTeam/greptimedb/blob/cfae276d37a65d14250185cd3d376a305078fdbd/src/datatypes/src/value.rs#L345